calculator.py

```diff
@@ -109,6 +114,7 @@ def start_calculator():
     salusion=take_expresion(args.f)
     cal= Calculator()
     print(cal.calculate(salusion))
+    print(cal.add.count_operation + cal.subtract.count_operation +cal.multiply.count_operation + cal.divide.count_operation) # pylint: disable=no-member
```

Comment:
Print it with some message.
`Add operations: <num>`
`Subtract operations: <num>`
...